src/Sulu/Component/Webspace/Loader/Exception/InvalidWebspaceLocalizationsException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace\Loader\Exception;
+
+use Sulu\Component\Webspace\Webspace;
+
+/**
+ * Thrown if some webspace locales are not used in any portal.
+ */
+class InvalidWebspaceLocalizationsException extends WebspaceException
```

Comment:
How about renaming this one to `WebspaceLocalizationNotUsedException`? `InvalidWebspaceLocalizationsException` could be any invalid configuration regarding localizations.